React/Base/RCTTouchHandler.m

```diff
@@ -150,6 +150,11 @@ - (void)_updateReactTouchAtIndex:(NSInteger)touchIndex
   reactTouch[@"locationX"] = @(touchViewLocation.x);
   reactTouch[@"locationY"] = @(touchViewLocation.y);
   reactTouch[@"timestamp"] =  @(nativeTouch.timestamp * 1000); // in ms, for JS
+  
+  if ([UIDevice currentDevice].systemVersion.floatValue >= 9) {
```

Comment:
This check should be more dynamic. Check if `forceTouchCapability` is defined (iOS 9+) and enabled.